src/microbe_stage/Microbe.cs

```diff
@@ -1015,4 +1026,15 @@ private void CountHexes()
 
         cachedHexCountDirty = false;
     }
+
+    private void SetDebugName()
+    {
+        if (Species == null!)
+        {
+            debugName = string.Empty;
+            return;
+        }
+
+        debugName = $"[{Species.Genus[0]}.{Species.Epithet.Substring(0, 4)}:{GetInstanceId()}]";
+    }
```

Comment:
Again, the same here. Also if the species is null I think it's better to just return the debug name without it, so it'd just be the instance id.

Reply:
Then I'll need more work to check if the species is properly applied and more runtime cost.

Reply:
I think it won't performance too much, should be very cheap to warrant to be run each frame. On a related note would nullables work in a string? Like I think it's more efficient to write `$"[Species?.name] ..."`.